doc/5-wave/README.md

```diff
@@ -110,6 +111,7 @@ input waveforms should match the size of the wavetable or unexpected results may
 - synthesizer type: selects the synthesis algorithm.
 - waveform displays.
 - **Wave 1**: selects input waveform.
+  - this will turn yellow to indicate that a Waveform macro is set; remove the macro to work with the synthesizer.
```

Comment:
Not necessarily. The waveform macro controls the first wave in that case, which may lead to interesting side effects.